nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -209,3 +209,5 @@
 - The `nettools` package (ifconfig, arp, mii-tool, netstat, route) is not installed by default anymore. The suite is unmaintained and users should migrate to `iproute2` and `ethtool` instead.
 
 - `sparkleshare` has been removed as it no longer builds and has been abandoned upstream.
+
+- The systemd target `kbrequest.target` is now unset by default, instead of being forcibly symlinked to `rescue.target`. In case you were relying on this behavior (Alt + ArrowUp on the tty causing the current target to be changed to `rescue.target`), you can restore it by setting `systemd.targets.rescue.aliases = [ "kbrequest.target" ];` or `systemd.targets.kbrequest.wants = [ "rescue.target" ];` in your configuration.
```

Comment:
nit:

```suggestion
- The systemd target `kbrequest.target` is now unset by default, instead of being forcibly symlinked to `rescue.target`. In case you were relying on this behavior (Alt + ArrowUp on the tty causing the current target to be changed to `rescue.target`), you can restore it by setting `systemd.targets.rescue.aliases = [ "kbrequest.target" ];` in your configuration.
```

Makes sense to just give the most direct equivalent here.

I do think this should go under the breaking changes section.